source/getting-started/use-cases/file-integrity.rst

```diff
@@ -62,7 +62,7 @@ The Wazuh FIM module supports various configuration options that enable effectiv
       :align: center
       :width: 80%
 
-   In alert fields, the *who-data* metadata shows that the user ``wazuh`` added the word ``Hello`` to the ``audit_docu.txt`` file using the ``Notepad.exe`` process.
+   In alert fields, the *who-data* metadata shows that the user ``wazuh`` modified the ``audit_docu.txt`` file using the ``Notepad.exe`` process.
```

Comment:
**question**: the text and image displayed a change in the text through the `syscheck.diff` field but it is not present on the new screenshot.

Reply:
Done